src/lib.rs

```diff
@@ -20,9 +20,6 @@ pub mod poll;
 pub mod rand;
 pub mod seek_hole;
 pub mod signal;
-// The rust musl libc implementation has defined msghdr.{__pad1, __pad2) as private, which causes
```

Comment:
The original sock_ctrl_msg.rs is copied from the crosvm project without modification. Is it preferred to separate changes for musl from the original import commit?

Reply:
I didn't realize it. You can leave it in separate commits in this case.

nit: can you also specify the CrosVM commit in your commit message. It might be useful in the future.

Reply:
done:)